src/lib/components/namespace-selector.svelte

```diff
@@ -10,7 +10,7 @@
   }
 </script>
 
-<Tooltip text="Select a Namespace">
+<Tooltip top text="Select a Namespace">
```

Comment:
Was this change intentional? And is this component currently being used anywhere atm? Wondering if we want to add translations.

Reply:
Ah yeah good call, I meant to remove this component because it's unused since Tideman implemented the new top-nav.